packages/server/api/src/app/pieces/piece-metadata-service/piece-metadata-service.ts

```diff
@@ -27,6 +27,7 @@ type ListParams = {
     orderBy?: PieceOrderBy
     searchQuery?: string
     suggestionType?: SuggestionType
+    premiumPieces?: string
```

Comment:
A better name would be includePremiumPiece and its type is boolean, i am thinking of prefer an enum over the boolean rule but i don't think it's applicable here